### PR TITLE
docs(router): add cookies example

### LINKED ISSUE #728: Docs: add example to API routes on accessing cookies
### Which scope/s are relevant/related to the feature request?

Docs

### Information

We added support for accessing the request event here. The PR has an example of how to access cookies in the server request.

https://github.com/analogjs/analog/pull/614

This example should be added to the docs in the API routes overview page.

https://github.com/analogjs/analog/blob/main/apps/docs-app/docs/features/api/overview.md

### Describe any alternatives/workarounds you're currently using

_No response_

### I would be willing to submit a PR to fix this issue

- [ ] Yes
- [ ] No

### PR DESCRIPTION
**Pull request by santoshyadavdev**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/analogjs/analog/blob/main/CONTRIBUTING.md#-commit-message-guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## Which package are you modifying?

- [ ] vite-plugin-angular
- [ ] vite-plugin-nitro
- [ ] astro-angular
- [ ] create-analog
- [x] router
- [ ] platform
- [ ] content
- [ ] nx-plugin
- [ ] trpc

## What is the current behavior?

Closes #728

## What is the new behavior?

Add example to API routes on accessing cookies and remove some unwanted imports 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: apps/docs-app/docs/features/api/overview.md [hunk 1]
```diff
@@ -176,6 +176,48 @@ export default defineEventHandler((event) => {
...
+
+### Reading cookies
+
+```ts
+//index.server.ts
+import { parseCookies } from 'h3';
+import { PageServerLoad } from '@analogjs/router';
+
+export const load = async ({ event }: PageServerLoad) => {
+  console.log('shipping');
```
Comment (brandonroberts) [inline apps/docs-app/docs/features/api/overview.md:210]:
```suggestion
```

File: apps/docs-app/docs/features/api/overview.md [hunk 2]
```diff
@@ -176,6 +176,48 @@ export default defineEventHandler((event) => {
...
+  const cookies = parseCookies(event);
+
+  console.log('test cookie', cookies['test']);
```
Comment (brandonroberts) [inline apps/docs-app/docs/features/api/overview.md:213]:
```suggestion
  console.log('products cookie', cookies['products']);
```

**Review by brandonroberts — CHANGES_REQUESTED** [review]

File: apps/docs-app/docs/features/api/overview.md [hunk 3]
```diff
@@ -176,6 +176,47 @@ export default defineEventHandler((event) => {
 });
 ```
 
+## Accessing Cookies
+
+Analog allows setting cookies reading them in your server side calls.
```
Comment (brandonroberts) [inline apps/docs-app/docs/features/api/overview.md:181]:
```suggestion
Analog allows setting and reading cookies in your server-side calls.
```

**Review by brandonroberts — APPROVED** [review]

**Comment by santoshyadavdev** [comment]
hey @brandonroberts should I rollback last commit and fix the message?

**Comment by brandonroberts** [comment]
@santoshyadavdev you can if you want to. It will get squashed into a single commit on merge either way

**Review by brandonroberts — APPROVED** [review]

**Pull request merged**